CRM/Upgrade/Incremental/sql/5.71.alpha1.mysql.tpl

```diff
@@ -1 +1,22 @@
 {* file to handle db changes in 5.71.alpha1 during upgrade *}
+
+-- Add name field, make frontend_title required (in conjunction with php function)
+{if $multilingual}
+  {foreach from=$locales item=locale}
+    UPDATE `civicrm_uf_group`
+    SET `frontend_title_{$locale}` = `title_{$locale}`, `name` = CONCAT(`title_{$locale}`, `id`);
```

Comment:
@eileenmcnaughton the problem is at the end here you have a `;` on this line which will stop the WHERE from being included